README.md

```diff
@@ -397,15 +397,15 @@ in [doc/plugins.md](doc/plugins.md#hooks).
 ### Limitations
 
 It is not possible to call Finit via signals or use `initctl` in any
-runparts or `/etc/rc.local` script.  This because Finit is single
```

Comment:
I believe this is correct English.

Reply:
I'm not a native speaker, but I still think a verb phrase is missing. "This is" would be the main clause, "because Finit is [...]" would be it's dependent clause.

Reply:
I must persist, I am quite sure this is correct English.  Please drop this change.